joplin/js/CreateContentModal/index.js

```diff
@@ -77,13 +85,28 @@ class CreateContentModal extends Component {
     this.setState({ topic: id });
   };
 
-  redirectToEditPage = () => {
-    this.writeToLocalStorage();
-    window.location.href = this.state.redirectUrl;
+  redirectToEditPage = id => {
+    window.location.href = `/admin/pages/${id}/edit/`;
```

Comment:
since we don't really reuse this redirect method, could we just kill the method and put the redirect line in the then block on line 104?